vite.config.js

```diff
@@ -0,0 +1,144 @@
+  const env = loadEnv(isDevMode ? 'testing' : mode, process.cwd(), '')
+  const proxyConfig = {
+    proxy: {
+      '/native': {
+        target: env.API_BASE_URL,
+        changeOrigin: true,
+      },
+      '/saml': {
+        target: env.API_BASE_URL,
+        changeOrigin: true,
+      },
+    },
+  }
+  return defineConfig({
+    define: {
+      global: 'window',
+      'process.env': env,
+      __DEV__: env.NODE_ENV !== 'production',
+    },
+    plugins: [
+      react(),
+      {
+        apply: 'build', // This plugin runs only when building (not serve)
+        name: 'treat-js-files-as-jsx',
+        async transform(code, id) {
+          if (!libsThatHaveJSFilesContainingJSX.some((lib) => id.includes(lib))) return null
+          return transformWithEsbuild(code, id, {
+            loader: 'jsx',
+            jsx: 'automatic',
+          })
+        },
+      },
+      createHtmlPlugin({
+        minify: true,
+        entry: '/src/index.tsx',
+        template: 'public/index.html',
+        inject: {
+          data: {
+            TITLE: packageJson.author.name,
+            DESCRIPTION: packageJson.description,
+            AUTHOR: packageJson.author.name,
+            TWITTER_SITE: packageJson.author.twitter,
+            META_NO_INDEX:
+              env.ENV !== 'production' ? `<meta name="robots" content="noindex" />` : '',
+            PUBLIC_URL: isProdMode ? env.APP_PUBLIC_URL : undefined,
+            IOS_APP_STORE_ID: env.IOS_APP_STORE_ID,
+            ANDROID_APP_ID: env.ANDROID_APP_ID,
+            APPS_FLYER_WEB_PUBLIC_KEY: env.APPS_FLYER_WEB_PUBLIC_KEY,
+            COMMIT_HASH: env.COMMIT_HASH,
+            VERSION: env.VERSION,
+          },
+        },
+      }),
+      // Put the Sentry vite plugin after all other plugins as specified in plugin's documentation
+      sentryVitePlugin({
+        url: 'https://sentry.passculture.team/',
+        org: 'sentry',
+        project: 'application-native',
+        authToken: env.SENTRY_AUTH_TOKEN, // locally from .env.local, otherwise will come from CI
+        release: {
+          uploadLegacySourcemaps: {
+            paths: ['./dist'],
+            ignore: ['node_modules'],
+          },
+          finalize: env.ENV !== 'testing',
+          cleanArtifacts: false,
+          name: `${packageJson.version}-web-${env.COMMIT_HASH}`,
+          dist: `${packageJson.build}-web-${env.COMMIT_HASH}`,
+          deploy: {
+            env: isDevMode ? 'development' : env.ENV,
+            name: isDevMode ? 'development' : env.ENV,
+            url: env.API_BASE_URL,
+          },
+        },
+      }),
+    ],
+    resolve: {
+      extensions: allExtensions,
+      alias: [
+        { find: /^((api|features|fixtures|libs|shared|theme|ui|web).*)/, replacement: '/src/$1' },
+        { find: 'react-native', replacement: 'react-native-web' },
+        {
+          find: 'react-native-email-link',
+          replacement: resolve(__dirname, 'src/libs/react-native-email-link'),
```

Comment:
les autres sont écrit plus simplement, pourquoi celui-ci nécessite `resolve(__dirname, ...)` ?

```suggestion
          replacement: '/src/libs/react-native-email-link',
```